Makefile

```diff
@@ -107,20 +108,20 @@ lint: pre-commit type-check 						## Run all linting
 .PHONY: coverage
 coverage:  											## Run the tests and generate coverage report
 	@echo "=> Running tests with coverage"
-	@$(ENV_PREFIX)pytest tests --cov -n auto
-	@$(ENV_PREFIX)coverage html
-	@$(ENV_PREFIX)coverage xml
+	@$(PDM) run pytest tests --cov -n auto
+	@$(PDM) run coverage html
+	@$(PDM) run coverage xml
 	@echo "=> Coverage report generated"
 
 .PHONY: test
 test:  												## Run the tests
 	@echo "=> Running test cases"
-	@$(ENV_PREFIX)pytest tests -n auto
```

Comment:
Why do we keep removing the `-n auto` from this? I only made a PR to add it back in yesterday and haven't seen any discussion about it.

Running the tests on multiple cores saves >30s per run.